homeassistant/components/fitbit/sensor.py

```diff
@@ -122,11 +124,35 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
     unit_fn: Callable[[FitbitUnitSystem], str | None] = lambda x: None
     scope: FitbitScope | None = None
 
+    @property
+    def is_tracker(self) -> bool:
+        """Return if the entity is a tracker."""
+        return FITBIT_TRACKER_SUBSTRING in self.key
+
+
+def _build_device_info(
+    config_entry: ConfigEntry, entity_description: FitbitSensorEntityDescription
+) -> DeviceInfo:
+    """Build device info for sensor entities info across devices."""
+    unique_id = cast(str, config_entry.unique_id)
+    if entity_description.is_tracker:
+        return DeviceInfo(
+            entry_type=DeviceEntryType.SERVICE,
+            identifiers={(DOMAIN, f"{unique_id}_tracker")},
+            translation_key="tracker",
+            translation_placeholders={"display_name": config_entry.title},
+        )
+    return DeviceInfo(
+        entry_type=DeviceEntryType.SERVICE,
+        identifiers={(DOMAIN, unique_id)},
+        name=config_entry.title,
```

Comment:
Name can be omitted if the device name is the same as the entry title as long as it is a primary device entry (which it is because of the entry type)

Reply:
Removed.